PeerioTechnologies/peerio-desktop
# feat: Quit menu item; swap sign out icon

#### Relevant info and issue/PR links 

https://app.clubhouse.io/peerio/story/10029/quit-menu-item
https://app.clubhouse.io/peerio/story/2582/don-t-close-app-on-window-close

Copy: https://github.com/PeerioTechnologies/peerio-icebear/pull/337

#### Testing instructions  

(Not on Mac. On Mac we don't have this...)

"Quit Peerio":

1. Open avatar menu.
2. Check that "Quit Peerio" is there.
3. Click it.
4. Peerio should quit.

"Update & Quit Peerio":

In dev console:

```js
require('./stores/updater-store').readyToInstall = true
```

Now check that menu item says "Update & Quit Peerio".

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
